lib/grape/middleware/base.rb

```diff
@@ -27,7 +29,21 @@ def call!(env)
         ensure
           after_response = after
         end
-        after_response || @app_response
+
+        response = after_response || @app_response
+        reflect_headers response
+        response
+      end
+
+      def reflect_headers(response)
+        return unless header.is_a?(Hash)
```

Comment:
Nope, there is a possibility of inserting `Rack::Accept::MediaType` into `@headers`.
See https://github.com/ruby-grape/grape/blob/master/lib/grape/middleware/versioner/header.rb#L69

However, aliasing seems reasonable to me. So I'd like to fix it.